packages/lib-classifier/src/store/Classification.js

```diff
@@ -36,15 +37,12 @@ const ClassificationMetadata = types.model('ClassificationMetadata', {
 
 const Classification = types
   .model('Classification', {
-    annotations: types.map(types.union({
-      dispatcher: (snapshot) => {
-        const snapshotType = getType(snapshot)
```

Comment:
`getType` can only be called on a tree node, not a snapshot, so I've had to remove this.